pkg/healthcheck/healthcheck_output.go

```diff
@@ -109,6 +236,11 @@ func runChecksTable(wout io.Writer, hc Runner) bool {
 		fmt.Fprintf(wout, "Status check results are %s\n", okStatus)
 	}
 
+	if output == ShortOutput {
```

Comment:
I think I'm -1 on adding this status count summary.  I don't think it's that useful, it feels odd that we include it for short but not table output, and capturing a statusCounter map into the prettyPrint closure smells bad.